src/main/java/uk/gov/hmcts/reform/bulkscanning/config/AccessTokenConfigurationProperties.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.bulkscanning.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.List;
+
+@ConfigurationProperties("accesstoken")
+public class AccessTokenConfigurationProperties {
```

Comment:
Now that I look at this I think of a shorter acceptable naming: `AccessTokenProperties`. But it's not that important really